engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version.html

```diff
@@ -37,9 +37,9 @@ <h3 translate>
         </a>
       </li>
 
-      <li ng-class="{active: isState('content-views.details.version.repositories')}">
-        <a ui-sref="content-views.details.version.repositories({versionId: version.id})">
-          <span translate>Repositories</span>
+      <li ng-class="{active: isState('content-views.details.version.yum')}">
+        <a ui-sref="content-views.details.version.yum({versionId: version.id})">
+          <span translate>Yum Content</span>
```

Comment:
What about `Yum Repositories` instead? This implies to me it's going to show me all yum content not just repositories.